ceph.spec.in

```diff
@@ -914,10 +846,8 @@ Group:		Development/Libraries/Python
 Requires:	librbd1 = %{_epoch_prefix}%{version}-%{release}
 Requires:	python%{python3_pkgversion}-rados = %{_epoch_prefix}%{version}-%{release}
 %{?python_provide:%python_provide python%{python3_pkgversion}-rbd}
-%if 0%{without python2}
-Provides:	python-rbd = %{_epoch_prefix}%{version}-%{release}
+Provides:	python-rbd < %{_epoch_prefix}%{version}-%{release}
```

Comment:
I think the Provides should get `=` and the Obsoletes should get `=`. Here, both have `<`?

Reply:
can we drop this change? as it's not quite right, and is a workaround. and instead use `zypper packages --orphaned` or `yum autoremove` to address the problem in https://tracker.ceph.com/issues/40099 ?

Reply:
> can we drop this change?

We might be able to replace it with something else, provided it has the same effect, but I guess that is unrelated to this PR and should be done elsewhere.

Reply:
> provided it has the same effect

i don't think we have an alternative with exactly the same effect. also, i don't think it makes sense to Provide python-* packages in a python3 env.

Reply:
>> i don't think we have an alternative with exactly the same effect. also, i don't think it makes sense to Provide python-* packages in a python3 env.

@smithfarm please do not reuse my comment for replying it. it's just confusing and i have to check the history for understanding the context.

i did read the tracker issue. i also suggested `zypper packages --orphaned` or `yum autoremove` to address the problem. what i am arguing is:

a python3 package should not provide `python-*`. which is just wrong. even if it is able to address the issue.